.github/CONTRIBUTING.md

```diff
@@ -1,32 +1,44 @@
-
-# Development
+# Contributing to SimpleITK
 
 We welcome contributions from the community!
 
-SimpleITK uses the [Git](http://git-scm.com/) distributed version control system. The main repository is hosted on [itk.org](https://itk.org/SimpleITK.git) and mirrored to [Github](https://github.com/SimpleITK/SimpleITK.git). We are currently using Github for pull requests, and issue tracking.
+SimpleITK uses the [Git](http://git-scm.com/) distributed version control system. The main repository is hosted on [Github](https://github.com/SimpleITK/SimpleITK.git) and mirred on [itk.org](https://itk.org/SimpleITK.git) and mirrored on the [Insight Software Consortium](https://github.com/InsightSoftwareConsortium/SimpleITK). For pull requests and issue tracking  Github is also used.
+
+There are many ways to contribute to SimpleITK:
+ - Report Bugs and Issues
+ - Improve SimpleITK Examples and Documentation
+ - Request new features
+ - Contribute code to fix bug or add new features.
+
+## Setup for development
+
+The following documentation is for an advanced contributor who is already familiar with git and GitHub workflow. The [ITK](https://github.com/InsightSoftwareConsortium/ITK) documentation may provide more details, or many online resource can orient you.
 
-# Contribute
+After you have clone and forked the SimpleITK repository run the following command from the root:
+```
+./Utilities/SetupForDevelopment.sh
+```
 
-To submit code to SimpleITK, simply follow the standard GitHub workflow to make a pull request. More information can be found [here](https://guides.github.com/activities/forking/) and [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962) regarding the workflow and getting your computer setup.
+This script will configure you local checkout with the `upstream` repository and your own fork. Additionally, local hooks and alias are added.
 
-New feature topics should be based on the `master` branch, while bugs fixes for the recent release should be based on the `release` branch. A topic should *never* be based on the `next` branch.
+## Workflow
 
-## Setup
+Submitting code to SimpleITK follows the standard GitHub workflow of make a pull request. More information can be found [here](https://guides.github.com/activities/forking/) and [here](https://guides.github.com/introduction/flow/) regarding the workflow and getting your computer setup.
```

Comment:
the standard GitHub workflow of make a pull request -> the standard pull request GitHub workflow.